src/components/Feature/Feature.tsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable valid-typeof */
+import Component from '../Component';
+import Subtitle from './Subtitle';
+import { Box, Container, Stack, Typography } from '@mui/material';
+
+export interface FeatureItem {
+  icon?: JSX.Element;
+  title: string;
+  subtitle: string;
+  linkTitle?: string;
+  url?: string;
+}
+
+export interface FeatureProps {
+  title: string;
+  items: FeatureItem[];
+  theme: 'dark' | 'light';
+}
+
+const Feature = ({ title, items, theme }: FeatureProps) => {
+  return (
+    <Component>
```

Comment:
Why don't use directly a `Container` avoiding a "wrapper element" ?

Reply:
i use wrapper element because before change of the last commit i used Showcase component of mui-italia. Should use i container component?

Reply:
yes, a container could be enough

Reply:
do it